### PR TITLE
Update packages

### PR DESCRIPTION
**Pull request by timonegk**

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -4,37 +4,29 @@ verify_ssl = true
...
-pyldap = "==3.0.0.post1"
-python-Levenshtein = "~=0.12"
+pygraphviz = "~=1.10"
+python-ldap = "~=3.4"
+python-Levenshtein = "~=0.20"
 python-creole = "~=1.4"
 python-magic = "~=0.4"
-pytz = "==2020.1"
 raven = "~=6.10"
-requests = "~=2.24"
```
Comment (ftsell) [inline Pipfile:36]: I still need requests for an upcoming PR for Vault integration

**Comment by timonegk** [comment]
This is ready for review now, the things I could test are working but there might still be some wrong urls. This should remain in staging for a while to be tested and probably wait for sentry to be deployed on master so that we can see when errors occur.

**Review by ftsell — COMMENTED** [review]
Thank you so much for this <3
I've kept an eye on this PR during its development and it looks good to me. I haven't properly reviewed it yet though ^^

**Review by ftsell — APPROVED** [review]
works as far as I could test it on my machine

**Pull request merged**